### PR TITLE
[ja] Translate tasks/debug-application-cluster/monitor-node-health into Japanese

### LINKED ISSUE #30870: [ja] Translate tasks/debug-application-cluster/monitor-node-health into Japanese
# This is a Feature Request

## What would you like to be added

Translate [tasks/debug-application-cluster/monitor-node-health](https://kubernetes.io/docs/tasks/debug-application-cluster/monitor-node-health/)  into Japanese.

## Why is this needed

This page has not been translated.

## Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by ptux**
resolve #30870

### CODE REVIEW

File: content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+reviewers:
+- ptux
+---
+
+<!-- overview -->
+
+*Node Problem Detector* は、ノードの健全性を監視し、報告するためのデーモンです。
+`Node Problem Detector`は`DaemonSet`として、あるいはスタンドアロンデーモンとして実行することができます。
+
+`Node Problem Detector`は様々なデーモンからノードの問題に関する情報を収集し、これらの状態を[NodeCondition](/docs/concepts/architecture/nodes/#condition)および[Event](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#event-v1-core)としてAPIサーバにレポートします。
```
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:13]:
```suggestion
`Node Problem Detector`は様々なデーモンからノードの問題に関する情報を収集し、これらの状態を[NodeCondition](/ja/docs/concepts/architecture/nodes/#condition)および[Event](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#event-v1-core)としてAPIサーバにレポートします。
```

File: content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+<!-- steps -->
+
+## 制限事項
+
+* Node Problem Detector は、ファイルベースのカーネルログのみをサポートします。
+  `journald` のようなログツールはサポートされていません。
```
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:25]:
```suggestion
  `journald`のようなログツールはサポートされていません。
```

File: content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+* Node Problem Detector は、カーネルの問題を報告するためにカーネルログフォーマットを使用します。
+  カーネルログフォーマットを拡張する方法については、 [Add support for another log format] (#support-other-log-format) を参照してください。
```
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:28]:
```suggestion
  カーネルログフォーマットを拡張する方法については、[Add support for another log format](#support-other-log-format) を参照してください。
```

File: content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+## ノード問題検出の有効化
+
+クラウドプロバイダーによっては、`Node Problem Detector`を{{< glossary_tooltip text="Addon" term_id="addons" >}} として有効にしている場合があります。
```
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:32]:
```suggestion
クラウドプロバイダーによっては、`Node Problem Detector`を{{< glossary_tooltip text="Addon" term_id="addons" >}}として有効にしている場合があります。
```

File: content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+### Addon podを使用してNode Problem Detectorを有効化します {#using-addon-pod}
+
+カスタムのクラスターブートストラップソリューションを使用していて、デフォルトの設定を上書きする必要がない場合は、`Addon Pod`を利用してデプロイをさらに自動化できます。
+`node-problem-detector.yaml`を作成し、制御プレーンノードの`Addon Pod`のディレクトリ`/etc/kubernetes/addons/node-problem-detector`に設定を保存します。
+
+## コンフィギュレーションを上書きします
+
+`Node Problem Detector`の Dockerイメージをビルドする際に、[default configuration(https://github.com/kubernetes/node-problem-detector/tree/v0.1/config)が埋め込まれます。
+
+しかし、[`ConfigMap`](/docs/tasks/configure-pod-container/configure-pod-configmap/) を使用して設定を上書きできます。
```
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:64]:
```suggestion
[`ConfigMap`](/ja/docs/tasks/configure-pod-container/configure-pod-configmap/) を使用することで設定を上書きすることができます。
```

File: content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+  "type": "temporary/permanent",
+  "condition": "NodeConditionOfPermanentIssue",
+  "reason": "CamelCaseShortReason",
+  "message": "regexp matching the issue in the kernel log"
+}
+```
+
+### カーネルログデバイスのパスの設定 {#kernel-log-device-path}
+
+ご使用のオペレーティングシステム（OS）ディストリビューションのカーネルログパスの場所をご確認ください。
```
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:127]:
```suggestion
ご使用のオペレーティングシステム(OS)ディストリビューションのカーネルログパスをご確認ください。
```

**Review by inductor — CHANGES_REQUESTED** [review]

File: content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
+---
+title: ノードの健康状態を監視します
```
[the remark this answers is not in the document]
Comment (ptux) [reply content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:2]: how about `健全性`?

[the remark this answers is not in the document]
Comment (inductor) [reply content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:2]: That kinda sounds like compliance perspective

[the remark this answers is not in the document]
Comment (riita10069) [reply content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:2]: @inductor 
Sorry for the late reply.

I think we often say `正常性`, as `ノードが正常か、異常かを表す名詞`.

I have a few more ideas below.

- `ノードのヘルスモニタリング`
- `ノードの状態を監視する`

@ptux 

`健全性` is also used in the content, perhaps it should be replaced with `正常性`.

[the remark this answers is not in the document]
Comment (ptux) [reply content/ja/docs/tasks/debug-application-cluster/monitor-node-health.md:2]: @inductor @riita10069 
Here are some use cases about「健全」.

content/ja/docs/concepts/workloads/pods/_index.md
> そして、スケジューラーが代わりのPodを健全なノード上に配置します。

And please google search for [kubernetes 健全].
You will find out that [健全性] is better.

content/ja/docs/setup/production-environment/tools/kubeadm/high-availability.md
>すべての健全なコントロールプレーンノードにトラフィックを分配します

**Comment by ptux** [comment]
@inductor 

This is a friendly reminder.

**Pull request merged**